StringCatalogEnum/README.md

```diff
@@ -40,3 +40,10 @@ It is expected for the new developers to proactively research unfamiliar tech st
 - `Makefile` and `make`.
 
 Hint: always ask questions if you are lost, just make sure you've done your research.
+
+## How to use SwiftLint before pushing
+
+1. Open your terminal
+2. Navigate to StringCatalogEnum
+3. Type "SwiftLint" and hit enter
```

Comment:
I guess you are trying to say:

```console
brew install swiftlint # install latest dependency, e.g. using homebrew here
cd XStringCatalog/StringCatalogEnum
swiftlint
```

Always use command over English if it's possible. You can also write in lines, which may be a bit better:

- Install latest dependency, e.g. using homebrew here: `brew install swiftlint`
- `cd XStringCatalog/StringCatalogEnum`
- `swiftlint`